src/components/Providers/helpers.js

```diff
@@ -0,0 +1,9 @@
+import { SETTINGS_DOCTYPE } from 'src/doctypes'
+
+export const saveAccountToSettings = async ({ client, setting, account }) => {
```

Comment:
Could we test this helper?

Reply:
quel genre de test vu la simplicité de la chose ???